k4e-agent-sti.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+set -e
+
+# The scripts performs the following steps on rhel-8.5:
+# * Install imagebuilder 
+# * Create rpms for yggdrasil and k4e-device-worker
+# * Publish rpms as yum repo to be consumed by imagebuilder
+# * Publish aarch64 rhel-image for edge-device
+
+# The script assumes the machine is registered via subscription-manager
+if ! subscription-manager refresh; then
+   echo  "The machine is not registered."
+   exit 1
+fi
+
+#---------------------------
+# Install osbuild components
+#---------------------------
+dnf install -y osbuild-composer composer-cli cockpit-composer bash-completion
```

Comment:
can we use the setup script instead?

Reply:
I'd rather this script not to depend on any other component. so it can simply be invoked by 
```
curl -s -L https://raw.githubusercontent.com/masayag/r4e/k4e-sti/k4e-agent-sti.sh | bash
```

Reply:
with this claim, we can have all scripts not depend one on another.
Someone might want to do:
curl https://..../r4e-setup.sh
and also
curl https://.../r4e-iso.sh
However, we are not a web app and therefore we do not support it.
I think that if you want to be like a web app then you should do what browser does: fetch all dependencies.
Just my 2 cents.

Reply:
With proposed curl it is easy to automate it without checking out entire project.

Reply:
I understand.
But, but it doesn't answer my comment.